docs/docs/docs/01-core/admin/01-ee/02-audit-logs.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Audit Logs
+slug: /settings/audit-logs
+description: Guide for Audit Logs
+tags:
+  - settings
+  - audit-logs
+---
+
+# Audit Logs
+
+## Summary
+
+Audit Logs provide a way to view the history of all user actions in a Strapi application. This includes actions related to entries (including publish actions), media (including its folders), users, login & logout of admin users, components, roles, and permissions.
+
+## Backend design
+
+The Audit Logs feature was built to take advantage of the eventHub. You can find the service and its core code at `packages/core/admin/ee/server/services`.
+
+### Subscribing to all events
+
+The Audit Logs feature adds a subscriber to the [EventHub](/strapi/event-hub), allowing it to listen to all events in the application. However, we don't save every event in the audit logs; we only save the default ones (see the defaultEvents array in the service file).
+
+### Retention days
+
+As the number of events in our Audit Logs can grow significantly, we run a daily job at midnight to delete logs that are older than the retention days defined.
+
+By default, the retention days are set to 90 days, but this value can be changed. For enterprise self-hosted projects, users can set a configuration variable (admin.auditLogs.retentionDays) and use that one.
+
+For cloud projects, the retention days are determined by the license. In a cloud project, users can set a custom retention days in the configuration, but this value cannot exceed the retention days defined by the license.
+
+### Audit Logs format
+
+Every Audit Log has the following format:
+
+```js
+const event = {
+  action, // Name of the event
+  date, // When the event happens
+  userId, // Id of the user that trigger the event
+  payload, // Extra info of the event
+};
+```
+
+To understand how we obtain this information, we need to know how we emit an event with the Event Hub. To emit an event, we use the following function: (To see more info about the EventHub, click [here](/strapi/event-hub))
+
+```js
+strapi.eventHub.emit('some.event', { meta: 'data' });
```

Comment:
What is `meta` and how does `meta` refer to the event definition you've just added above?

Reply:
Yeah - or maybe even `strapi.eventHub.emit(name: string, payload: Pick<Event, 'payload'>);` if you'd like to express it using TS?